src/backend/distributed/transaction/transaction_management.c

```diff
@@ -190,14 +190,14 @@ InCoordinatedTransaction(void)
 
 
 /*
- * CoordinatedTransactionShouldUse2PC() signals that the current coordinated
+ * Use2PCForCoordinatedTransaction() signals that the current coordinated
```

Comment:
I think the reason for `should` has been noted here. For read-only queries we do not use 2PC even if we are in coordinated transaction, that's why we left it as `shouldUse`. That's why I have a slight preference to revert this change, though if you think strongly about it we can leave this as-is (or come up with a better name?)

Reply:
I miss the "should" as well, but I feel like names of functions that perform a particular action should start with a verb, so in that sense I felt it was a slight improvement.